source/compiler/sc5.c

```diff
@@ -197,6 +198,10 @@ static char *warnmsg[] = {
 /*239*/  "literal array/string passed to a non-const parameter\n"
 };
 
+static char *noticemsg[] = {
+/*001*/  "did you mean \"%s\"?"
```

Comment:
Would `"; did you mean \"%s\"?\n"` be better?  This would be more in-line with the convention in all the other error/warning messages.